src/App.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import { BrowserRouter as Router,Routes, Route, Link } from 'react-router-dom';
+import Operation from './operations';
+export default function App(){
+  const [is, setIs] = useState(true)
+  return(
+
+    <Router>
+           <div className="App">
+            <ul className="App-header">
+              <li  className={`${is ? "" : "hidden"} flex h-[100vh] flex-col justify-center`}>
```

Comment:
since you only have one alternative, you can use a logical AND operator here
```suggestion
              <li  className={`${!is  && "hidden"} flex h-[100vh] flex-col justify-center`}>
```